src/test/java/org/example/action/ManagedCookieAction.java

```diff
@@ -17,62 +17,24 @@
 
 import io.fusionauth.http.Cookie;
 import org.primeframework.mvc.action.annotation.Action;
-import org.primeframework.mvc.action.result.annotation.Status;
 import org.primeframework.mvc.scope.annotation.ManagedCookie;
 
 /**
- * @author Daniel DeGroff
+ * @author Spencer Witt
  */
 @Action
-@Status
 public class ManagedCookieAction {
```

Comment:
Moved the existing code to `ManagedCookieScopesAction` and updated this one to be more like the test pages for encrypted and compressed cookies.